### DISCUSSION TITLE
Facial Recognition models - whats the difference?

### DISCUSSION THREAD

**Discussion by DrSpaldo** [immich-app/immich#4081] [Q&A]
I see that you can currently select between two different facial recognition models - buffalo_s & buffalo_l 

I was wondering what the difference was?

**Comment by thariq-shanavas** [comment]
bufallo_l is a larger model with much better accuracy. bufallo_s is smaller and faster.
For an apples-to-apples comparison on how much better the larger model is, see here: https://pypi.org/project/insightface/0.7/
